pantheon-bundle/frontend/src/app/searchBeta.tsx

```diff
@@ -0,0 +1,570 @@
+              {this.state.products.map((data) => (
+                <SimpleListItem key={data.id}>
+                  <Checkbox label={data.name} aria-label="uncontrolled checkbox" id={data.id} />
+                </SimpleListItem>
+              ))}
+            </SimpleList>
+
+          </ExpandableSection> */}
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+    const drawerContent = (
+      <React.Fragment>
+        <ExpandableSection toggleText="Modules" className="pf-c-title" isActive={true} isExpanded={modulesIsExpanded} onToggle={this.onModulesToggle}>
+          <SearchResults
+            contentType="module"
+            keyWord={this.state.inputValue}
+            repositoriesSelected={this.state.repositoriesSelected}
+            productsSelected={this.state.productsSelected}
+            userAuthenticated={this.props.userAuthenticated}
+            filters={this.state.filters}
+          />
+
+        </ExpandableSection>
+        <br />
+        <ExpandableSection toggleText="Assemblies" className="pf-c-title" isActive={true} isExpanded={assembliesIsExpanded} onToggle={this.onAssembliesToggle}>
+          <SearchResults
+            contentType="assembly"
+            keyWord={this.state.inputValue}
+            repositoriesSelected={this.state.repositoriesSelected}
+            productsSelected={this.state.productsSelected}
+            userAuthenticated={this.props.userAuthenticated}
+            filters={this.state.filters}
+          />
+
+        </ExpandableSection>
+      </React.Fragment>
+    );
+
+    const statusMenuItems = [
+      <SelectOption key="statusDraft" value="draft" label= "Draft" />,
+      <SelectOption key="statusPublished" value="released" label="Published" />
+    ];
+
+    const contentTypeMenuItems = [
+      <SelectOption key="ctypeConcept" value="CONCEPT" label="Concept" />,
+      <SelectOption key="ctypeProcedure" value="PROCEDURE" label="Procedure" />,
+      <SelectOption key="ctypeReference" value="REFERENCE" label="Reference" />
+    ];
+
+    const toggleGroupItems = (
+      <React.Fragment>
+        <ToolbarItem>
+          <Button variant="tertiary" aria-expanded={isExpanded} onClick={this.onClick} icon={<FilterIcon />} />
+        </ToolbarItem>
+        <ToolbarItem>
+          <InputGroup>
+            <SearchInput
+              name="textInput"
+              id="textInput"
+              placeholder="Find by name"
+              type="search"
+              aria-label="search input"
+              onChange={this.onInputChange}
+              onClear={this.onInputClear}
+              value={inputValue}
+            />
+            <Button variant={ButtonVariant.control} aria-label="search button for search input">
+              <SearchIcon />
+            </Button>
+          </InputGroup>
+        </ToolbarItem>
+        <ToolbarGroup variant="filter-group">
+          <ToolbarFilter
+            chips={filters.status}
+            // deleteChip={this.onDelete}
```

Comment:
do you need the `onDelete` function here?

Reply:
Currently, users have two options to remove filter chips

-  "clear all filters" 

-  uncheck items from the dropdown

ideally, we enable deleteChip prop for ToobarFilter component. This gives user a third option to remove chip. I created a jira ticket to capture the effort: CCS-3975